src/router/index.js

```diff
@@ -102,6 +103,16 @@ router.beforeEach((to, from, next) => {
   // clear all toasts whenever the route changes
   toast.clear();
 
+  // show auto logout toast
+  if (
+    to.name == "Login" &&
+    from.name != undefined &&
+    from.name != "Login" &&
+    to.params["userClickedLogout"] == "false"
```

Comment:
would be good to instead use this so that it doesn't break when `userClickedLogout` is not passed some time in the future (as we might forget this then):
```python
to.params.get("userClickedLogout")
```

Reply:
Javascript doesn't have a `get` method
And I checked with not passing `userClickedLogout`, as it's a string,`to.params["userClickedLogout"] == "false"` just gives the output as a boolean `false` so it won't break